src/ceph-disk/ceph_disk/main.py

```diff
@@ -2443,7 +2451,7 @@ def populate(self):
         LOG.debug('Mounting lockbox ' + str(" ".join(args)))
         command_check_call(args)
         write_one_line(path, 'osd-uuid', self.args.osd_uuid)
-        self.create_key()
+        self.create_key(self.args.cluster)
```

Comment:
the self.create_key method should use self.args.cluster: there is no need to pass it as an argument